api/src/main/resources/Reference_Application_Diagnoses-11.xml

```diff
@@ -1,240 +1,240 @@
-  <concept concept_id="160199" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2011-06-09 12:44:50.0" version="" changed_by="1"  uuid="160199AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-  <concept concept_id="160200" retired="false" datatype_id="4" class_id="4" is_set="false" creator="1" date_created="2011-06-09 12:46:35.0" version="" changed_by="1" date_changed="2011-06-11 16:23:10.0"  uuid="160200AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+  <concept_datatype concept_datatype_id="2" name="Coded" hl7_abbreviation="CWE" description="Value determined by term dictionary lookup (i.e., term identifier)" creator="1" date_created="2004-02-02 00:00:00.0" retired="false" retired_by="1" uuid="8d4a48b6-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_datatype concept_datatype_id="4" name="N/A" hl7_abbreviation="ZZ" description="Not associated with a datatype (e.g., term answers, sets)" creator="1" date_created="2004-02-02 00:00:00.0" retired="false" retired_by="1" uuid="8d4a4c94-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_datatype concept_datatype_id="10" name="Boolean" hl7_abbreviation="BIT" description="Boolean value (yes/no, true/false)" creator="1" date_created="2004-08-26 00:00:00.0" retired="false" retired_by="1" uuid="8d4a5cca-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_class concept_class_id="4" name="Diagnosis" description="Conclusion drawn through findings" creator="1" date_created="2004-02-02 00:00:00.0" retired="false" retired_by="1" uuid="8d4918b0-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_class concept_class_id="5" name="Finding" description="Practitioner observation/finding" creator="1" date_created="2004-03-02 00:00:00.0" retired="false" retired_by="1" uuid="8d491a9a-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_class concept_class_id="10" name="ConvSet" description="Term to describe convenience sets" creator="1" date_created="2004-03-02 00:00:00.0" retired="false" retired_by="1" uuid="8d492594-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_class concept_class_id="11" name="Misc" description="Terms which don&apos;t fit other categories" creator="1" date_created="2004-03-02 00:00:00.0" retired="false" retired_by="1" uuid="8d492774-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_class concept_class_id="12" name="Symptom" description="Patient-reported observation" creator="1" date_created="2004-10-04 00:00:00.0" retired="false" retired_by="1" uuid="8d492954-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept_class concept_class_id="13" name="Symptom/Finding" description="Observation that can be reported from patient or found on exam" creator="1" date_created="2004-10-04 00:00:00.0" retired="false" retired_by="1" uuid="8d492b2a-c2cc-11de-8d13-0010c6dffd0f"/>
+  <concept concept_id="48" retired="false" datatype_id="2" class_id="4" is_set="false" creator="1" date_created="2004-01-01 00:00:00.0" version="" changed_by="1" date_changed="2012-10-09 21:18:04.0" retired_by="1" uuid="48AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
@dkayiwa  These changes are intentional. I mean the contents of this file was generated using the mdsbuilder. If you notice, most of these changes are involving the addition of a new attribute ("retired_by="1""). Don't know whether this is needed. I have tried removing these small changes and running the build locally. It still fails. Apart from this, it is worth noting that there are a few genuine additions to this file as well.